data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v2/validator/GetNewBlock.java

```diff
@@ -159,20 +167,4 @@ public void handleRequest(RestApiRequest request) throws JsonProcessingException
                     .map(AsyncApiResponse::respondOk)
                     .orElseThrow(ChainDataUnavailableException::new)));
   }
-
-  private static SerializableOneOfTypeDefinition<BeaconBlock> getBlockSchemaDefinition(
```

Comment:
This cleanup basically allows us to filter the capella milestone when listing types of blinded blocks... doing our own handling here was going to mean a lot of duplication of filters.